### DISCUSSION TITLE
Fastlane match doesn't create provisioning profile for certain appID

### DISCUSSION THREAD

**Discussion by zahs** [fastlane/fastlane#19854] [Q&A]
hi,

I am trying to renew my adhoc/appstore cert and profiles.\
I am using nuke to revoke them firstly.
afterwards running a match command to create cert and profiles.
`match_nuke(
            type: type,
            app_identifier: appIdentifier,
            username: username,
            git_user_email: git_user_email
        )`

`match(
        type: type,
        app_identifier: appIdentifier,
        profile_name: profile_name,
        generate_apple_certs: true,
        template_name: template_name,
        username: username,
        git_user_email: git_user_email
    )`
    
 for one of my profiles it gives the following error
    
` {
  "errors" : [ {
    "id" : "id...,
    "status" : "409",
    "code" : "ENTITY_ERROR",
    "title" : "There is a problem with the request entity",
    "resultCode" : 8530,
    "detail" : "There was an error generating the provisioning profile   Please try again.  If the problem persists, please contact Apple Developer Program Support. https://developer.apple.com/support"
  } ]
}`

I am able to create this profile manually using already generated certificate.
What can be the issue?

**Comment by lego-sharat** [comment]
were you able to find any resolution to this. I am facing the same issue
